### ISSUE TITLE
bcftools view -a fails on variants with 100% missingness

### ISSUE THREAD

**Issue by agilly** [samtools/bcftools#461]
Hi, just a little bug we found trying to pinpoint an error.

When running on a VCF that contains the following line:

```
chr22   18204688        .       A       G       70.38   .       AC=0;AF=1;AN=0
```

that contains only missing data, `bcftools view -a` fails with the following message:

```
bcftools: vcfutils.c:632: bcf_remove_allele_set: Assertion `nori==nG_ori' failed.
Aborted
```

Cheers,

A

**Comment by pd3** [comment]
Can you please show also the AC, AF and AN header lines? What version of bcftools are you running? I was not able to reproduce the problem with 1.3 and this file:

         ##fileformat=VCFv4.3
         ##INFO=<ID=AC,Number=A,Type=Integer,Description="">
         ##INFO=<ID=AF,Number=A,Type=Float,Description="">
         ##INFO=<ID=AN,Number=1,Type=Integer,Description="">
         #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
         1	14000	.	A	C	.	PASS	AC=0;AF=1;AN=0

**Comment by kousikbioinfo** [comment]
Hi, 
I'm getting this problem in both bcftools versions (1.2 and 1.3)

```
##fileformat=VCFv4.2
##INFO=<ID=AC,Number=A,Type=Integer,Description="Allele count in genotypes, for each ALT allele, in the same order as listed">
##INFO=<ID=AF,Number=A,Type=Float,Description="Allele Frequency, for each ALT allele, in the same order as listed">
##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
#CHROM POS ID  REF ALT QUAL    FILTER  INFO
chr1   19024085        .       GT      G       61.73   .       AC=0;AF=1;AN=0
```

**Error:** 
`bcftools: vcfutils.c:616: bcf_remove_alleles: Assertion 'nori==nG_ori' failed
`


Cheers,
Kousik

**Comment by mcshane** [comment]
This should be closed via samtools/htslib@ee47258.
